constants.py

```diff
@@ -64,6 +59,23 @@ class TraceData:
         VARTYPE: pd.StringDtype(),
     }
 
+class TypeHintData:
+    FILENAME = "Filename"
+    CLASS_MODULE = "ClassModule"
+    CLASS = "Class"
+    FUNCNAME = "FunctionName"
+    LINENO = "LineNo"
+    CATEGORY = "Category"
+    VARNAME = "VarName"
+    VARTYPE_MODULE = "TypeModule"
+    VARTYPE = "Type"
+
```

Comment:
Consider moving all columns into one class and all schema into one class.

For example: constants.Schemas.TypeHintData, constants.Schemas.TraceData, constants.Schemas.Columns.FILENAME, ...